src/co_nutate.jl

```diff
@@ -0,0 +1,101 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _co_nutate(jd::T, ra::T, dec::T) where {T<:AbstractFloat}
+    d_psi, d_eps = nutate(jd)
+    eps = obliquity(jd)
```

Comment:
Yeap. Didn't see anyway around it except modifying `obliquity` to return d_psi and d_eps too. But `nutate` already does that, so let it be.